mobile/app/src/main/java/org/exarhteam/iitc_mobile/IITC_WebViewPopup.java

```diff
@@ -145,9 +151,28 @@ public void onDismiss(DialogInterface dialog) {
 
     private void openDialogPopup() {
         if (mDialog.isShowing()) return;
+
+        // Retrieve display metrics from the context
+        DisplayMetrics displayMetrics = new DisplayMetrics();
+        ((WindowManager) mIitc.getSystemService(Context.WINDOW_SERVICE)).getDefaultDisplay().getMetrics(displayMetrics);
+
+        // Calculate 80% of screen height and 90% of width
+        int height = (int) (displayMetrics.heightPixels * 0.8);
+        int width = (int) (displayMetrics.widthPixels * 0.9);
+
+        // Apply the calculated height and width to the dialog
+        WindowManager.LayoutParams layoutParams = new WindowManager.LayoutParams();
+        layoutParams.copyFrom(mDialog.getWindow().getAttributes());
+        layoutParams.height = height;
+        layoutParams.width = width;
```

Comment:
Yeah, this will make the popups look a bit better, but we should imho use

a) height/width with `match_parent`
b) use horizontalMargin/verticalMargin to specify "The horizontal margin, as a percentage of the container's width, between the container and the widget." - https://www.dre.vanderbilt.edu/~schmidt/android/android-4.0/out/target/common/docs/doc-comment-check/reference/android/view/WindowManager.LayoutParams.html